### PR TITLE
APIGOV-27632 - add advanced search description

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add the advanced search result page details

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 6d0e945 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-437.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by cbordier** [comment]
@lbadenhop you can start review that one.

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 1]
```diff
@@ -70,6 +72,8 @@ Use the search bar to find products by keywords with a match in:
 * Documentation (markdown only)
 * Categories
 
+Once you hit Enter, the corresponding matching product are displayed. You can use the [filters](#refine-your-findings-with-filters) to farther refine the search or use the [Advanced search](#advanced-search) to create more complex queries.
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:75]: After testing, I don't think you need to press enter to get the search results. They are displayed as you type. I suggest:

"The corresponding matching product are displayed. You can use the [filters](#refine-your-findings-with-filters) to refine the search or use the [Advanced search](#advanced-search) to create more complex queries."

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 2]
```diff
@@ -80,12 +84,26 @@ When viewing the product list, refine your search with Advanced Search.
...
-2. Once you are satisfied with your criteria, click **Search**. The corresponding matching products are displayed.
+2. Once you are satisfied with your criteria, click **Search**. You are redirected to the [Advanced search result page](#advanced-search-result-page).
 
 Click **Reset** to remove the selected criteria and refresh the product list to the one that is visible by the user.
 
 Click **Basic Search** to return to the Basic Search bar at any time.
 
+#### Advanced search result page
+
+When reaching that page, the search result is split to help you understand where the result were found:
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:95]:
```suggestion
To help you understand where your advanced search results were found, the results are displayed by product, product resources, and product documentation.
```

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 3]
```diff
@@ -80,12 +84,26 @@ When viewing the product list, refine your search with Advanced Search.
...
+
+* in the product itself
+* in the product resources
+* in the product documentation
+
+By default the All tab is visible and regroup all result in the same place. If there is no match for one type, the corresponding section remain collapsed and cannot be expanded. This view is limited to the first 25 results. You can see more details by using the **View all** buttons or by navigating via the tab menu (All -> Products - Resources - Documents).
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:101]:
```suggestion
By default, the *All* tab displays the advanced search results. If no match is found in an area of search (product/product resources/product documentation), then the corresponding results section will not expand to display the results. If results are found, the view will be limited to the first 25. Click **View all** or navigate the tab menu (All > Products/Resources/Documents) to view all results.
```

File: content/en/docs/manage_marketplace/consumer_experience/searching.md [hunk 4]
```diff
@@ -80,12 +84,26 @@ When viewing the product list, refine your search with Advanced Search.
...
+
+Each result, the name is a link that redirect you to the appropriate page in the product details page.
+
+The breadcrumb at the top of the page enables to navigate from where you came (Home page / Browse product page).
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/consumer_experience/searching.md:105]: {{< alert title="Tip" color="secondary" >}}Navigate back to your previous location by using the breadcrumb at the top of the page{{< /alert >}}

**Comment by lbadenhop** [comment]
@cbordier Please see my comments. I did not merge comments in case you want to ignore them.

File: content/en/docs/manage_marketplace/consumer_experience/searching.md (hunk 1 above)
Comment (cbordier) [reply content/en/docs/manage_marketplace/consumer_experience/searching.md:75]: We changed that behavior. Now you will have to type Enter to get to the result.

**Comment by cbordier** [comment]
@lbadenhop, I reviewed, commit your changes. It remains the 1st item where we do changed the behavior.

**Pull request merged**